test/countries/test_malaysia.py

```diff
@@ -164,6 +200,24 @@ def test_Malaysia_Wikipedia(self):
                 1,
                 1,
             ],
+            date(2021, FEB, 14): [
```

Comment:
The formatting makes this test looking crazy. Some sort of a moving platform (The Platform? :] ).
We'll need to refactor this when time comes.

Reply:
Yes, it was something, editing this, believe me :)

Reply:
I will repeat my request that we finally move to 120 character lines. The 1980s restriction to 80 can lead to some very dizzying code! @dr-prodigy

Reply:
The 120 limit may be an overkill in this particular case. I'd say 100 is more appropriate for python-holiday in general.

The formatting issue can be fixed without changing the line length limit. You just need to remove the last comma in each weird looking list.

And by refactoring I meant replacing the way the holidays table is represented at the moment with something more readable/efficient (my main priority for tests is readability).

I hope this clarifies my previous comment.

Reply:
This is how it'd look like with max length 100 (technically, 99) https://github.com/dr-prodigy/python-holidays/pull/804